netdata-installer.sh

```diff
@@ -476,20 +476,27 @@ trap build_error EXIT
 # -----------------------------------------------------------------------------
 
 build_libmosquitto() {
+  CFLAGS=
+  LDFLAGS=
+  CXXFLAGS=
+
   if [ "$(uname -s)" = Linux ]; then
-    run env CFLAGS= CXXFLAGS= LDFLAGS= make -C "${1}/lib"
+    run env CFLAGS="$CFLAGS" CXXFLAGS="$CXXFLAGS" LDFLAGS="$LDFLAGS" make -C "${1}/lib"
```

Comment:
The code here deliberately removes the CFLAGS because otherwise the build breaks in certain configurations. Why is this PR attempting to put them back in?

Reply:
It doesn't. If you look a few lines above it sets them appropriately.

Reply:
Onlines 4790-481